controllers/kyma_controller/moduletemplate_test.go

```diff
@@ -40,108 +25,9 @@ func expectManifestSpecDataEquals(kymaName, value string) func() error {
 	}
 }
 
-func expectManifestSpecNotContainsCredSecretSelector(kymaName string) func() error {
-	return func() error {
-		createdKyma, err := GetKyma(ctx, controlPlaneClient, kymaName, "")
-		if err != nil {
-			return err
-		}
-		for _, module := range createdKyma.Spec.Modules {
-			moduleInCluster, err := GetManifest(ctx, controlPlaneClient, createdKyma, module)
-			if err != nil {
-				return err
-			}
-			if moduleInCluster.Spec.Config.CredSecretSelector != nil {
-				return ErrContainsUnexpectedCredSecretSelector
-			}
-			installImageSpec := extractInstallImageSpec(moduleInCluster.Spec.Install)
-
-			if installImageSpec.CredSecretSelector != nil {
-				return ErrContainsUnexpectedCredSecretSelector
-			}
-		}
-		return nil
-	}
-}
-
-func expectManifestSpecContainsCredSecretSelector(kymaName string) func() error {
-	return func() error {
-		createdKyma, err := GetKyma(ctx, controlPlaneClient, kymaName, "")
-		if err != nil {
-			return err
-		}
-		for _, module := range createdKyma.Spec.Modules {
-			moduleInCluster, err := GetManifest(ctx, controlPlaneClient, createdKyma, module)
-			if err != nil {
-				return err
-			}
-			var emptyImageSpec v1beta2.ImageSpec
-			if moduleInCluster.Spec.Config != emptyImageSpec {
-				if err := expectCredSecretSelectorCorrect(&moduleInCluster.Spec.Config); err != nil {
-					return fmt.Errorf("config %v is invalid: %w", &moduleInCluster.Spec.Config, err)
-				}
-			}
-
-			installImageSpec := extractInstallImageSpec(moduleInCluster.Spec.Install)
-			if err := expectCredSecretSelectorCorrect(installImageSpec); err != nil {
-				return fmt.Errorf("install %v is invalid: %w", installImageSpec, err)
-			}
-		}
-		return nil
-	}
-}
-
 func extractInstallImageSpec(installInfo v1beta2.InstallInfo) *v1beta2.ImageSpec {
 	var installImageSpec *v1beta2.ImageSpec
 	err := yaml.Unmarshal(installInfo.Source.Raw, &installImageSpec)
 	Expect(err).ToNot(HaveOccurred())
 	return installImageSpec
 }
-
-func expectCredSecretSelectorCorrect(installImageSpec *v1beta2.ImageSpec) error {
-	if installImageSpec.CredSecretSelector == nil {
-		return fmt.Errorf("image spec %v does not contain credSecretSelector: %w",
-			installImageSpec, ErrNotContainsExpectedCredSecretSelector)
-	}
-
-	value, found := installImageSpec.CredSecretSelector.MatchLabels[credSecretLabel]
-	Expect(found).To(BeTrue())
-	Expect(value).To(Equal(credSecretValue))
-	return nil
-}
-
-var _ = Describe("ModuleTemplate.Spec.descriptor not contains RegistryCred label", Ordered, func() {
```

Comment:
Why you deleted this two test?

Reply:
They are valid testing

Reply:
Since they were reading the Manifest.Spec.Config. Now that we won't write anything to the config, the tests don"t make sense.

Reply:
No wait. You're right.

Reply:
We don't have the credselector in the raw install

Reply:
no, that's not true, the `CredSecretSelector` used in `manifest.Spec.Install` you can find here: https://github.com/kyma-project/lifecycle-manager/blob/main/internal/manifest/spec_resolver.go#L139

Reply:
ok fair point.